ports/nrf/boards/s140_7.2.0.ld

```diff
@@ -1,4 +1,4 @@
 /* GNU linker script for s140 SoftDevice version 6.1.1 */
 
-_sd_size = 0x00026000;
+_sd_size = 0x00027000;
```

Comment:
No size change for s132?

Reply:
No, not for the S132,  it's still the same address.  Only the S140 has the address changed.